docs/compilers/CSharp/Compiler Breaking Changes - DotNet 9.md

```diff
@@ -0,0 +1,22 @@
+# This document lists known breaking changes in Roslyn after .NET 8 all the way to .NET 9.
+
+## .editorconfig values no longer support trailing comments
+
+***Introduced in Visual Studio 2022 version [TBD]***
```

Comment:
This is just a reminder to fix this prior to merge.

Reply:
❓ Do we know what the value here is going to be?